packages/frontend/src/notebook/notebook_editor.css

```diff
@@ -23,7 +23,284 @@
 
 .notebook .placeholder {
     color: darkgray;
-    & .lucide {
-        color: darkgray;
+}
+
+.overlay {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: fixed;
+    opacity: 0;
+    visibility: hidden;
+    transition: all 0.3s ease;
+    z-index: 1000;
+}
```

Comment:
Is there any reason why much of this CSS was not included in `landing_page.css`? That seems like a much more natural spot for it and I would avoid splitting `.overlay` selector over two files.

Reply:
I see. Then they should probably use different css classes. The two overlays seem completely different and having them use the same class is going to cause confusion in the future.